### ISSUE TITLE
Unmaintained

### ISSUE THREAD

**Issue by untitaker** [untitaker/mysteryshack#74]
I don't use remoteStorage anymore, at all.

If somebody wants to keep maintaining this, I'd be happy to make an organization.

**Comment by anderspitman** [comment]
Hi @untitaker, I was curious why you're no longer using remoteStorage? The ecosystem as a whole seems to have gone a little stale, which is sad because I think it's a great idea and something like it should exist. I've been considering building something similar, but much simpler. Just curious if you had any thoughts on self-hosted storage in general?

**Comment by untitaker** [comment]
@anderspitman it's a fine protocol, but there is a lack of abstractions over it. My experience with remotestorage.js (the library) hasn't been particularly pleasurable. I wish somebody built a database and ORM on top of remotestorage for JS or any language that compiles to wasm.

I also have some minor concerns that I raised in the spec repo but nothing insurmountable. The main issue is that there's really no resources going into polishing that would be direly needed.

I use nextcloud for my private stuff.

**Comment by anderspitman** [comment]
Thanks @untitaker. Does NextCloud include any way to delegate permissions to 3rd-party apps? That seems like the key feature of something like RS to me.

**Comment by untitaker** [comment]
No, but what is there is much more polished. That's more revelant in practice

**Comment by pataquets** [comment]
@untitaker Thanks for sharing.
Check out https://adoptoposs.org . It might be useful.
HTH.

**Comment by emembrives** [comment]
Hello, I updated mysteryshack to more up-to-date dependencies (see https://github.com/emembrives/mysteryshack/). If you want to merge back, feel free to do so. If your proposal of transferring the maintenance still applies, I would be happy to discuss.

**Comment by raucao** [comment]
@untitaker Wondering if you missed the last comment here perhaps?

**Comment by untitaker** [comment]
I would prefer it if people explicitly switched over to the maintained fork since it's now a different entity maintaining it. I prefer that over handing over maintainership. I am happy to add a link to the readme.

This service is so old I think somebody should rewrite it on top of a more modern framework. I might do that sometime.
